### PR TITLE
added new theme

### PR DESCRIPTION
**Pull request by OddExtension5**
theme named: oddextension

### CODE REVIEW

**Comment by anuraghazra** [comment]
Hey @OddExtension5 thanks for adding new Theme, can you please show a screenshot of it?

File: themes/index.js [hunk 1]
```diff
@@ -71,6 +71,12 @@ const themes = {
     text_color: "f8f8f2",
     bg_color: "282a36",
   },
+  oddextension: {
+    title_color: "EE0BF5",
```
Comment (anuraghazra) [inline themes/index.js:75]: Let's prefer lower case values :D also i just noticed in the top of the file there are two Upper case values in bg_color (default, default_repocard) can you please make then lowercase too?

Comment (OddExtension5) [reply themes/index.js:75]: Here is the view of my recent theme: used your profile for example

  Profile Stats<br>

Comment (anuraghazra) [reply themes/index.js:75]: Not going to lie, tbh i did not really liked the color scheme :disappointed_relieved:

Comment (OddExtension5) [reply themes/index.js]: Okay, np. Let me try another combination then but before that can you tell me which part you don't like most. So, I will work on that to bring a nice scheme

Comment (anuraghazra) [reply themes/index.js]: The colors does not really match up, that pink with grayish background kinda hurting the eyes & green icons with brown text is also not matching up, maybe try material colors

**Comment by anuraghazra** [comment]
Hi @OddExtension5 thanks for opening the PR. 

Note that if this theme is exclusively for your personal use then instead of adding it to our theme collection you can use card [customization options](https://github.com/anuraghazra/github-readme-stats#customization) 

Read our [contribution guidelines](https://github.com/anuraghazra/github-readme-stats/blob/master/CONTRIBUTING.md) for more info

**Comment by anuraghazra** [comment]
Closing this for inactivity, feel free to open another one if you want :) thanks!

**Pull request closed without merging**